internal/api/shared/schemas/testdata/samples/invalid_unwanted_query_field.json

```diff
@@ -0,0 +1,40 @@
+{
+    "kind": "Panel",
+    "spec": {
+        "display": {
+            "name": "simple first chart"
+        },
+        "plugin": {
+            "kind": "FirstChart",
+            "spec": {
+                "queries": [ 
+                    {
+                        "kind": "TimeSeriesQuery",
+                        "spec": {
+                            "plugin": {
+                                "kind": "CustomGraphQuery",
+                                "spec": {
+                                    "datasource": {
+                                        "kind": "CustomDatasource",
+                                        "name": "MyCustomDatasource"
+                                    },
+                                    "custom": true,
+                                    "aaaaaa": "I shouldnt be there"
+                                }
+                            }
+                        }
+                    }
+                ],
+                "a": "yes",
+                "b": {
+                    "c": [
+                        {
+                            "e": "up",
+                            "f": "the up metric"
+                        }
+                    ]
+                }
+            }
+        }
+    }
+}
```

Comment:
missing a line at the end of the file. It's the case for most of the new json files